paper-drawer-panel.html

```diff
@@ -493,6 +497,23 @@
           },
 
           /**
+           * The selector for the element that should receive focus when the drawer is open.
+           */
+          drawerFocusSelector: {
+            type: String,
+            value:
+                'a[href]:not([tabindex=\'-1\']),'+
+                'area[href]:not([tabindex=\'-1\']),'+
+                'input:not([disabled]):not([tabindex=\'-1\']),'+
+                'select:not([disabled]):not([tabindex=\'-1\']),'+
+                'textarea:not([disabled]):not([tabindex=\'-1\']),'+
+                'button:not([disabled]):not([tabindex=\'-1\']),'+
+                'iframe:not([tabindex=\'-1\']),'+
```

Comment:
If you want this part to look cleaner, you can use double quotes (`"`) instead of escaping single quotes.

TIL that `iframe`s are focusable.